crates/bevy_pbr/src/lib.rs

```diff
@@ -67,14 +68,18 @@ pub const SHADOWS_HANDLE: HandleUntyped =
     HandleUntyped::weak_from_u64(Shader::TYPE_UUID, 11350275143789590502);
 pub const PBR_SHADER_HANDLE: HandleUntyped =
     HandleUntyped::weak_from_u64(Shader::TYPE_UUID, 4805239651767701046);
+pub const PBR_PREPASS_SHADER_HANDLE: HandleUntyped =
+    HandleUntyped::weak_from_u64(Shader::TYPE_UUID, 9407115064344201137);
 pub const PBR_FUNCTIONS_HANDLE: HandleUntyped =
     HandleUntyped::weak_from_u64(Shader::TYPE_UUID, 16550102964439850292);
 pub const SHADOW_SHADER_HANDLE: HandleUntyped =
     HandleUntyped::weak_from_u64(Shader::TYPE_UUID, 1836745567947005696);
 
 /// Sets up the entire PBR infrastructure of bevy.
 #[derive(Default)]
-pub struct PbrPlugin;
+pub struct PbrPlugin {
+    pub prepass_enabled: bool,
```

Comment:
This needs a link to an entry point to read about prepasses.

Reply:
I didn't add much docs here because this wouldn't really be easily accessible unless you are already actively looking for it. The entry point is going to be the PrepassSettings on the camera. But I could link to it from here.

Reply:
Yep, just a link to that is helpful. I like to think of the API docs as a web: even peripheral stuff should link back into the core.